environments/production/inventory.ini

```diff
@@ -276,13 +276,13 @@ es_data
 
 
 [couch0]
-10.202.10.19 hostname=couch0-production ec2=yes
+10.202.40.211 hostname=couch0-production ec2=yes
 
 [couch1]
-10.202.11.235 hostname=couch1-production ec2=yes
+10.202.41.83 hostname=couch1-production ec2=yes
 
 [couch2]
-10.202.12.181 hostname=couch2-production ec2=yes
+10.202.42.85 hostname=couch2-production ec2=yes
```

Comment:
Were these intended to be part of this PR?

Reply:
Ooops, I didn't, but it's a change that's also on master from #2337. This happened when I ran `aws-fill-inventory`, and did reflect the correct IPs. (So while it's unrelated to the other changes and didn't have a place in this PR or that commit per se, and was a mistake, there's nothing I need to undo.)